deployability/modules/allocation/README.MD

```diff
@@ -0,0 +1,444 @@
+
+  Example
+
+  ```bash
+  python3 -m workflow_engine modules/workflow_engine/examples/dtt1-agents-poc.yaml
+  ```
+
+  > Note The command execution can also be mediated through Jenkins.
+
+#### Manual execution of the Allocation module
+
+If one wishes to execute the allocation module without installing the workflow engine, they can proceed by using the launcher ([module/allocation/main.py](main.py)):
+
+1. Create
+
+  While in wazuh-qa/deployability
+
+- Local deployment (Vagrant)
+
+  ```bash
+  python3 modules/allocation/main.py --action create --provider '{{ vagrant }}' --size '{{ large }}' --composite-name '{{ composite-name }}' --inventory-output '{{ inventory }}' --track-output '{{ track }}'
+
+  ```
+
+  Example:
+  ```bash
+  python3 modules/allocation/main.py --action create --provider vagrant --size large --composite-name linux-ubuntu-22.04-amd64 --inventory-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/inventory.yaml" --track-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/track.yaml"
+  ```
+
+- AWS deployment
+
+  ```bash
+  python3 modules/allocation/main.py --action create --provider '{{ aws }}' --size '{{ large }}' --composite-name '{{ composite-name }}' --inventory-output '{{ inventory }}' --track-output '{{ track }}' --label-termination-date '{{ termination-date }}' --label-team  '{{ team }}'
+
+  ```
+
+  >Note: In the case of AWS it is mandatory to define 2 arguments that are not necessary for Vagrant, --label-termination-date and --label-team.
+   --label-termination-date: This argument allows you to define the date on which the machine can be deleted. The allowed values are 1d (where the 1 refers to the number of days the machine is needed) or with the following format "2024-03-20 21:00:00"
+   --label-team: This argument allows you to set the team that owns the VM to be able to track it. The valid options are: 'qa', 'core', 'framework', 'devops', 'frontend', 'operations', 'cloud', 'threat-intel', 'marketing', 'documentation'
+
+
+
+  Example:
+  ```bash
+  python3 modules/allocation/main.py --action create --provider aws --size large --composite-name linux-ubuntu-22.04-amd64 --inventory-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/inventory.yaml" --track-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/track.yaml" --label-termination-date "2024-03-20 21:00:00"  --label-team devops
+  ```
+
+2. Delete
+
+  While in wazuh-qa/deployability
+
+  ```bash
+  python3 modules/allocation/main.py --action delete --track-output '{{ track }}'
+
+  ```
+
+  Example:
+  ```bash
+  python3 modules/allocation/main.py --action delete --track-output "/tmp/dtt1-poc/agent-linux-ubuntu-22.04-amd64/track.yaml"
+  ```
+
+  >Note: The --track-output argument is mandatory for the delete action because this file contains all information of the VM that will be destroyed
+
+3. Arguments
+
+  - --provider
+    This argument allows us to choose on which platform we will deploy our VM. The allowed values are aws or vagrant.
```

Comment:
Same, please emphasize the allowed values, at first I thought they were the service/software names (apply this to all the document)

Reply:
Done a81aec813